modules/utxo-lib/test/fixtures_thirdparty/nodes/dogecoin-1.14.5/src/test/data/sighash.json

```diff
@@ -0,0 +1,503 @@
+[
+	["raw_transaction, script, input_index, hashType, signature_hash (result)"],
```

Comment:
out of curiosity, how did you generate this file?

Reply:
It's from [dogecoin github repo](https://github.com/dogecoin/dogecoin/blob/master/src/test/data/sighash.json).

Reply:
!!! I didn't know that's where you find it - awesome!